src/tutorials/annotation.md

```diff
@@ -128,7 +161,8 @@ annotations in Excel or another tool.
 Instructions for doing this are available in the
 [Spreadsheet templates tutorial](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates).
 
-This process and templates make it convenient to provide basic descriptions, as well as HED tags for your dataset events.
+This process and templates make it convenient to provide basic
+descriptions, as well as HED tags for your dataset events.
 
 A [HED annotation quickstart](https://hed-examples.readthedocs.io/en/latest/HedAnnotationQuickstart.html)
```

Comment:
A better link is:
https://www.hed-resources.org/en/latest/HedAnnotationQuickstart.html

Reply:
Updated in latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).